### ISSUE TITLE
TensorRT Execution [ONNXRuntimeError] : 6 : RUNTIME_EXCEPTION : Exception during initialization

### ISSUE THREAD

**Issue by L-Reichardt** [microsoft/onnxruntime#10914]
**Describe the bug**
My Tensorflow -> ONNX converted model runs on CPU, but not executing through TensorRT or CUDA (CUDA separate issue #10894 ). Tensorflow model itself works fine on GPU. Found no related Issue or solution related to this Issue.
Models and logs found here [Google Drive](https://drive.google.com/drive/folders/1cIoRX7tYtX1Ze0Dx_0roT4IA3pUZjjlp?usp=sharing) (file format blocked by github)

**Urgency**
Urgent: by Monday 21.03.2022

**System Information**
- Linux Ubuntu 20.04
- ONNX Runtime installed from binary:
- ONNX Runtime version: gpu 1.10.0
- Python version: 3.8.10
- Visual Studio version (if applicable): n.a.
- GCC/Compiler version (if compiling from source): n.a.
- CUDA/cuDNN version: 10.04, 8.2.4
- TensorRT Version: 8.2.1, also tested with all newer versions
- GPU model and memory: RTX3060 12 GB

**To Reproduce**
- Attached Tensorflow model converted to attached ONNX Model
- Conversion with Opset 13

**Expected behavior**
- Failure to run model through ONNX runtime with TensorRT execution

**Screenshots**

**Additional context**
